### PR TITLE
Use eslint

### PR DESCRIPTION
**Pull request by SimenB**
I've recently joined the bandwagon that is standard.
As you used semicolons i picked `semistandard`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,27 +15,26 @@
...
     "rimraf": "^2.3.4",
+    "semistandard": "^6.1.2",
     "should": "^6.0.3"
   },
   "scripts": {
     "clean": "rimraf coverage/",
-    "lint": "jscs index.js test/ && jshint index.js test/",
-    "pretest": "npm run lint",
-    "test": "mocha",
+    "cover": "istanbul cover _mocha",
```
Comment (SimenB) [inline package.json]: this and the ones further down is npm alphabetizing

File: index.js [hunk 2]
```diff
@@ -44,15 +41,15 @@ var cssLintPlugin = function(options) {
   var rcLoader = new RcLoader('.csslintrc', options, { loader: 'async' });
 
   // Build a list of all available rules
-  csslint.getRules().forEach(function(rule) {
+  csslint.getRules().forEach(function (rule) {
```
Comment (lazd) [inline index.js]: Spaces before braces doesn't match my personal coding style, can we change that?

File: index.js [hunk 3]
```diff
@@ -44,15 +41,15 @@ var cssLintPlugin = function(options) {
...
     ruleset[rule.id] = 1;
   });
 
-  return es.map(function(file, cb) {
+  return es.map(function (file, cb) {
     if (file.isNull()) return cb(null, file); // pass along
-    if (file.isStream()) return cb(new error('gulp-csslint: Streaming not supported'));
+    if (file.isStream()) return cb(new gutil.PluginError('gulp-csslint: Streaming not supported'));
```
Comment (lazd) [inline index.js]: +1 this is the way it should be.

File: index.js [hunk 4]
```diff
@@ -77,30 +73,30 @@ var cssLintPlugin = function(options) {
...
       c.red('[') +
       (
-        typeof message.line !== 'undefined' ?
-          c.yellow( 'L' + message.line ) +
-          c.red(':') +
-          c.yellow( 'C' + message.col )
+      typeof message.line !== 'undefined' ?
+        c.yellow('L' + message.line) +
+        c.red(':') +
+        c.yellow('C' + message.col)
```
Comment (lazd) [inline index.js]: This DOES match my personal coding style -- I don't like to see spaces inside of braces :P

**Comment by lazd** [comment]
Besides the coding style differences and the lack of mergability, it's looking good! Please rebase and we'll merge it, I can play with the settings afterwards.

**Comment by SimenB** [comment]
This is using `standard`, so no changing rules... It's just `eslint` under the hood though, so I can update this using extension instead :smile:

**Comment by lazd** [comment]
@SimenB yeah I'd rather have it match the coding style of my other projects and gulp plugins. Please do!

**Comment by SimenB** [comment]
@lazd Updated!

**Comment by SimenB** [comment]
@lazd  I see you changed some formatting now, updated this to follow those conventions

**Comment by lazd** [comment]
This is looking good, I see several formatting fixes that make me happy, and it passes now. Merging!

**Pull request merged**